scripts/core/editor3/components/media/MediaBlock.tsx

```diff
@@ -22,6 +22,25 @@ function getTranslationForAssignRights(value) {
     }
 }
 
+interface IOwnProps {
+    block: ContentBlock;
+    contentState: ContentState;
+}
+
+interface IReduxStateProps {
+    readOnly: boolean;
+    showTitle: boolean;
+}
+
+interface IDispatchProps {
+    cropImage(entityKey, entityData, options): void;
+    removeBlock(blockKey): void;
+    changeCaption(entityKey, newCaption, field): void;
```

Comment:
can you add functions' prop types here?

Reply:
Functions have arguments, not props ;) I've added props from the data in proptypes and it wasn't that detailed. I didn't run that feature, just improved props so I don't know the types of arguments. I did add a few of the ones that were easy to understand from usage.